http/src/server/background.rs

```diff
@@ -93,13 +93,27 @@ impl BackgroundHttp {
                 };
 
                 runtime.block_on(async move {
+                    let builder = match hyper::Server::try_bind(&addr) {
```

Comment:
We reaaaaally want `bind` to fail if `hyper::Server::try_bind` fails. That's the whole point of returning an error in `bind`.

Reply:
:man_facepalming: